libr/anal/fcn.c

```diff
@@ -231,7 +231,7 @@ static int try_walkthrough_jmptbl(RAnal *anal, RAnalFunction *fcn, int depth, ut
 	anal->iob.read_at (anal->iob.io, ptr, jmptbl, MAX_JMPTBL_SIZE);
 	for (offs = 0; offs < MAX_JMPTBL_SIZE; offs += sz) {
 		ut64 jmpptr = 0;
-		r_mem_copyendian ((ut8*)&jmpptr, jmptbl + offs, sz, !anal->big_endian);
+		memcpy ((ut8*)&jmpptr, jmptbl + offs, sz);
```

Comment:
why not just use ut64 *jmpptr = (ut64*)jmptbl and then index with [i]? to avoid the unnecesary memcpyes